fuse_examples/classification/cmmd/runner.py

```diff
@@ -298,57 +250,37 @@ def run_eval(paths: dict, eval_common_params: dict):
 
     # run
     results = evaluator.eval(ids=None,
-                     data=os.path.join(paths["inference_dir"], eval_common_params["infer_filename"]),
+                     data=os.path.join(cfg.paths.inference_dir, cfg.infer.infer_filename),
                      metrics=metrics,
-                     output_dir=paths['eval_dir'])
+                     output_dir=cfg.paths.eval_dir)
 
     return results
-######################################
-# Run
-######################################
-
 
-if __name__ == "__main__":
-    # allocate gpus
-    if NUM_GPUS == 0:
-        TRAIN_COMMON_PARAMS['manager.train_params']['device'] = 'cpu'
```

Comment:
Did you move it to the configuration file?

Reply:
yes